Data/EnumMap.hs

```diff
@@ -1,3 +1,5 @@
+{-# LANGUAGE CPP, DataKinds, FlexibleContexts, FlexibleInstances,
```

Comment:
Could I ask you to add a comment that all these extensions are only needed for `Whoops`? Thanks!